test/unit/annotation_spec.js

```diff
@@ -1295,12 +1313,12 @@ describe('annotation', function() {
         { ref: lineRef, data: lineDict, }
       ]);
 
-      var annotation = AnnotationFactory.create(xref, lineRef, pdfManagerMock,
-                                                idFactoryMock);
-      var data = annotation.data;
-      expect(data.annotationType).toEqual(AnnotationType.LINE);
-
-      expect(data.lineCoordinates).toEqual([1, 2, 3, 4]);
+      AnnotationFactory.create(xref, lineRef, pdfManagerMock,
+                idFactoryMock).then((annotation) => {
```

Comment:
Indent this like you did here: https://github.com/mozilla/pdf.js/pull/9417/files#diff-04ad2b4c022ab1eae0d58931b8fe0b7dR74